src/content/reference/hi/p5/log.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: log
+module: नींव
+submodule: नींव
+file: src/core/reference.js
+description: >
+    <p>आपके ब्राउज़र के वेब कंसोल पर एक संदेश प्रिंट करता है। p5 का उपयोग करते समय, आप <a href='/reference/p5/print'>print</a> और 
+    <a href='/reference/p5/console/log'>console.log</a> का परस्पर उपयोग कर सकते हैं। </p>
+
+    <p>आप जिस ब्राउज़र का उपयोग कर रहे हैं उसके आधार पर कंसोल अलग-अलग तरीके से खोला जाता है। यहां 
+    <a href="https://developer.mozilla.org/en-US/docs/Tools/Web_Console/Opending_the_Web_Console">फ़ायरफ़ॉक्स</a>, 
+    <a href="https) में कंसोल खोलने के तरीके के लिंक दिए गए हैं ://developers.google.com/web/tools/chrome-devtools/open">Chrome</a>, 
+    <a href="https://docs.microsoft.com/en-us/microsoft-edge/devtools -guide/console">एज</a>, और 
+    <a href="https://support.apple.com/en-ca/guide/safari/sfri20948/mac">सफारी</a>। 
+    <a href='https://editor.p5js.org/'>ऑनलाइन p5 एडिटर</a> के साथ कंसोल सीधे कोड एडिटर के नीचे पेज में एम्बेड किया जाता है।</p>
+
+    <p> <a href="https://developer.mozilla.org/en-US/docs/Web/API/Console/log">MDN प्रविष्टि</a> से कंसोल विधि लॉग() आउटपुट करता है वेब कंसोल पर संदेश. संदेश एक एकल <a href="/reference/p5/string">स्ट्रिंग</a> हो सकता है (वैकल्पिक प्रतिस्थापन मानों के साथ), या यह कोई एक या अधिक JavaScript <a href="/reference/p5/ हो सकता है object">वस्तु</a>.</p>
+line: 512
+isConstructor: false
+itemtype: method
+alt: This example does not render anything
+example:
+  - |-
+
+    <div class='norender'>
+    <code>
+    let myNum = 5;
+    console.log(myNum); // prints 5 to the console
+    console.log(myNum + 12); // prints 17 to the console
+    </code>
+    </div>
+class: console
+params:
+  - name: message
+    description: |
+      <p>संदेश जिसे आप कंसोल पर प्रिंट करना चाहते हैं</p>
```

Comment:
Instead of prompt maybe something else can be used?

Reply:
Can you please elaborate on this?